build/build-and-unit-tests.yml

```diff
@@ -9,8 +9,16 @@ jobs:
 - job: BuildAndUnitTests${{ parameters.configuration }}
   displayName: Build and run unit tests - ${{ parameters.configuration }}
   condition: succeeded()
+  templateContext:
+              outputs:
```

Comment:
What's the intended indent on the outputs section? It seems far more indented that I would expect

Reply:
Fixed indentation